restfiles/perf-test-search.restfile

```diff
@@ -0,0 +1,124 @@
+# run as follows
+# env base_url=... rester perf-test-search.restfile --count 100 --stats
+mode: random
```

Comment:
Rester can be run in a repeat loop and with `mode: random` it will pick a random request from the list.

The request list below is just 20 random search terms.